### ISSUE TITLE
Zapzap is not trying to reconnect when it's launched on a system without an active internet connection

### ISSUE THREAD

**Issue by wokawoka** [rafatosta/zapzap#213]
Zapzap is not trying to reconnect when it's launched on a system initially without an active internet connection that gets connected in a later moment.
Thank you

**Comment by rafatosta** [comment]
@wokawoka 
Could you test before releasing the update?

Remove what has installed and install the package below:

`flatpak install zapzap.flatpak`

https://github.com/rafatosta/zapzap/releases/download/t5.3.6/zapzap.flatpak

I check the connection and, if you do not have internet, try to reconnect every 5 seconds.

 If everything is ok I will send the update to FLATHUB. Thanks.

**Comment by wokawoka** [comment]
I quickly tested it and it works here even with KDE autostart.
I will test it more extensively later tonight when back home, but for now it seems it's working very well.
Thank you!

**Comment by wokawoka** [comment]
Hello, unfortunately zapzap from time to time stll gets stuck when the system is connecting (or reconnecting after standby) to flaky internet connections or while the connection is not yet completely established (laptop temporarily connected without internet access for some seconds).

Quite often I realize that zapzap is not connected to whatsapp web even if my laptop is online since hours (likely because when I resumed the laptop from standby the connection was sub-optimal).
Today I did a fresh boot of the laptop in a coffee shop and zapzap launched at start but got stuck even if the lap went online.

Maybe there's a way to improve that. In this case I'd be happy to test any change.
Thank you very much for the useful job you are doing with this app.
